application/templates/static_site/export/measure_export.html

```diff
@@ -7,398 +7,361 @@
-                    </div>
-                    <div class="column-full">
-                        <h3 class="heading-small">Summary</h3>
-                        <div class="bullets spaced-out">
-                            {{ dimension.summary | render_markdown }}
-                        </div>
-                    </div>
-                    <div class="column-one-third"
-                         style="{% if not dimension.table and not dimension.chart %}display: none;{% endif %}">
-
-                    </div>
-                {% endfor %}
-            {% endif %}
-        </div>
-
-        <div class="column-full">
-
-            <h1 class="heading-large">Methodology</h1>
 
             <div style="padding-bottom: 10px">
-                <div class="heading-small">Methodology</div>
-                {% if measure_page.methodology %}
-                    <div id="methodology">{{ measure_page.methodology | render_markdown }}</div>
+                <div id="publication-release-date" class="heading-small">Source data publication date</div>
+                {% if measure_page.primary_data_source and measure_page.primary_data_source.publication_date %}
+                    <div id="publication-release-date-value">{{ measure_page.primary_data_source.publication_date }}</div>
                 {% else %}
                     <div>[...]</div>
                 {% endif %}
             </div>
 
             <div style="padding-bottom: 10px">
-                <div class="heading-small">Suppression rules and disclosure control</div>
-                {% if measure_page.suppression_and_disclosure %}
-                    <div id="suppression-and-disclosure">{{ measure_page.suppression_and_disclosure | render_markdown }}</div>
+                <div id="publication-frequency" class="heading-small">How often is the source data published?</div>
+                {% if measure_page.primary_data_source and measure_page.primary_data_source.frequency_of_release.description %}
+                    <div id="publication-frequency-value" >{{ measure_page.primary_data_source.frequency_of_release.description }}</div>
                 {% else %}
                     <div>[...]</div>
                 {% endif %}
             </div>
 
             <div style="padding-bottom: 10px">
-                <div class="heading-small">Rounding</div>
-                {% if measure_page.estimation %}
-                    <div>{{ measure_page.estimation | render_markdown }}</div>
+                <div id="type-of-data" class="heading-small">Type of data</div>
+                {% if measure_page.primary_data_source and measure_page.primary_data_source.type_of_data %}
+                    <div id="type-of-data-value">{{ measure_page.primary_data_source.type_of_data | join_enum_display_names(' and ') }} data</div>
                 {% else %}
                     <div>[...]</div>
                 {% endif %}
             </div>
 
             <div style="padding-bottom: 10px">
```

Comment:
Would probably be nice to factor these inline styles out into a class. Shouldn't be much work?

Also, the divs that use these styles seem massively repeated. Would it be worth making that a template fragment that could be included for each form field? The outer structure looks identical - but maybe the innards change too much for it to be worth it. Your call?

Reply:
@samuelhwilliams yeah it's a weird template. I decided to 'just' do the reordering for now, and not get tempted into much refactoring. I still don't totally understand how this page is used.